src/main/java/spireMapOverhaul/zones/gremlinTown/powers/BellowPower.java

```diff
@@ -0,0 +1,36 @@
+
+package spireMapOverhaul.zones.gremlinTown.powers;
+
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.abstracts.AbstractSMOPower;
+import spireMapOverhaul.zones.gremlinTown.GremlinTown;
+
+import static spireMapOverhaul.util.Wiz.adp;
+import static spireMapOverhaul.util.Wiz.applyToSelf;
+
+public class BellowPower extends AbstractSMOPower {
+    public static final String POWER_ID = SpireAnniversary6Mod.makeID("Bellow");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public BellowPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, GremlinTown.ID, PowerType.BUFF,false, owner, amount);
+    }
+
+    @Override
+    public void updateDescription() {
+        description = DESCRIPTIONS[0].replace("{0}", String.valueOf(amount));
+    }
+
+    @Override
+    public void onSpecificTrigger() {
```

Comment:
Why not just check enemy intent instead of that weird patch?

Reply:
Is there a hook with the correct timing so I don't have to patch?  The question of when to check the intent is relevant due to enemies such as slimes and writhing mass.

Reply:
end turn? Hmm, yeah I can see the issue